composer.json

```diff
@@ -16,16 +16,15 @@
     ],
     "require": {
         "php": "^5.3.9|^7.0",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder": "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3|~3.0|~4.0",
+        "symfony/finder": "~2.3|~3.0|~4.0",
         "psr/log": "~1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.5",
-        "phpunit/php-code-coverage": "~2.2",
-        "symfony/console": "~2.3|~3.0",
-        "symfony/expression-language": "~2.6|~3.0",
-        "symfony/templating": "~2.3|~3.0",
+        "phpunit/phpunit": "~4.5|~5.7",
```

Comment:
you may want to allow `^6.0` too

Reply:
First I tried using `^6.0`, but that phpunit version switches to using PHP namespaces and I thought it would require too many changes in the test files, this is why I stuck with 5.7.

Reply:
Switching to the namespaced classes is probably a good idea. But could be done in a different PR IMO.

Reply:
You can use 4.8.35 or higher on PHP 5.5 which provides a forward compatible layer for the namespaced classes.